dtreeviz/utils.py

```diff
@@ -125,6 +125,18 @@ def myround(v,ndigits=2):
     return format(v, '.' + str(ndigits) + 'f')
 
 
+def _normalize_class_names(class_names, nclasses):
```

Comment:
_function_name(), single underscore is a hint for private/internal usage. Since this method is used externally, I guess we should remove the _

Reply:
Ah.  I was using it to indicate for use inside of our library, although not necessarily limited to the module. I just wanted people to avoid using it outside dtreeviz.

Reply:
I don't really have a preference; whatever you think is best